### ISSUE TITLE
Question about pretrained models w.r.t InferSent and ESIM

### ISSUE THREAD

**Issue by adorable-mx** [jind11/TextFooler#47]
Hi, thanks for your code. Since I haven't found pretrained models for InferSent and ESIM, I trained both models by myself. However, I cannot reproduce the accuracy same as the data you provided. Could you provide details of training and validation? Or share the pretrained models?

**Comment by jind11** [comment]
I am so sorry that this project was done two years ago and I have moved from my school to my company now so it is not possible to retrieve the trained models any more. You may use the trained models from other sources, such as this one: https://github.com/facebookresearch/InferSent, https://github.com/coetaur0/ESIM.

**Comment by sunersheng** [comment]
Hello，is the argument "target-model-path" in attack_nli.py where the pretrained bertmodel exists? I can't load the model?

**Comment by jind11** [comment]
Yes, target-model-path should be the path to the model that has been fine-tuned on the corresponding dataset.

**Comment by sunersheng** [comment]
Please tell me do you have the pretrained model that you have finetuned? I just want to use the tool to generate some adversary sentences.

**Comment by jind11** [comment]
![Screen Shot 2021-12-10 at 9 54 41 AM](https://user-images.githubusercontent.com/16600322/145619560-f92d2307-3082-4dde-9d63-f10c1c9cefbe.png)
As shown in this screenshot, we did provide the fine-tuned models in the links.
